### PR TITLE
PHP 8.2 support

### PR DESCRIPTION
**Pull request by dada-amater**
#### What's this PR do?
Podpora pro PHP 8.2 a cleanup

### CODE REVIEW

**Comment by dada-amater** [comment]
Task linked: [AMA-5734 Upgrade na PHP 8.2 (backend)](https://app.clickup.com/t/2439381/AMA-5734)

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 vendor
-composer.lock
```
Comment (AdamLutka) [inline .gitignore:2]: Lock do klihoven nepatri.

**Review by AdamLutka — CHANGES_REQUESTED** [review]

**Review by AdamLutka — CHANGES_REQUESTED** [review]
Proc revert? Vyhod commit z vetve ne?

**Comment by dada-amater** [comment]
> Proc revert? Vyhod commit z vetve ne?

Jo, nechtel jsem behem CR menit historii.

**Review by AdamLutka — APPROVED** [review]

**Pull request merged**